src/Pages/ItemDetail/Components/Board.scss

```
@@ -0,0 +1,73 @@
+@import "../../../Styles/common.scss";
+
+.Board {
+  margin-bottom: 100px;
+  .menu-header {
+    margin-bottom: 10px;
```

가독성을 위해 css 선택자 간에는 한 줄 띄어주세요!
```suggestion
  margin-bottom: 100px;
  
  .menu-header {
    margin-bottom: 10px;
```

---

넵! 수정하겠습니다!